docs/user_guide/getting_started.rst

```diff
@@ -28,6 +28,27 @@ installed. There are two options for this:
 Install boa
 ===========
 
+If you need to create a new environment:
```

Comment:
I think we should change this to just instruct users to make a new environment; I'm worried if we make it sound optional they will either be confused or they'll install in their base environment or some other environment that already has a bunch of other stuff in it.

Reply:
That's fair, but I was also thinking about cases where they already have an environment for their model. How about something like "if you don't already have a dedicated condo environment for your model..."?

Reply:
That sounds good. At some point we'll need to add some other documentation about cases where there might be environment conflicts between a model and boa, and then we could link it on this page